### DISCUSSION TITLE
STLs links seem broken

### DISCUSSION THREAD

**Discussion by jhiggason** [StephenCarlson/MiniHawk-VTOL#35] [Q&A]
Hi there- I went to check out the naceles and the links appear to be broken.

https://github.com/StephenCarlson/MiniHawk-VTOL/blob/master/stl-SourceFiles/MH7_Nacelle_SideA.stl

=)  Thanks for all your hard work on this. A buddy and I are trying something similar with a much bigger design for SAR applications.  (Except we are going to tailsitter route)

**Comment by jhiggason** [comment]
In fact it seems a lot of your STL links are broken currently.

**Reply by StephenCarlson** [reply]
Ouch! Yeah, everything in the README needs to be fixed! Being a grad student means I'm always pushing small changes whenever I have a minute. I'll fix the names now. It should hopefully be easy to resolve what the link intended to point to, in this case I changed the name scheme to better match a parent_relationship / variant-relationship metric. The underscore represents a parent/child link, and the dash represents a variation. I'm still trying to propagate the naming change to all the parts.
